composer.json

```diff
@@ -25,21 +25,26 @@
         "ext-PDO": "*",
         "ext-redis": "*",
         "php": "^7.2",
-        "Krinkle/intuition": "^1.1",
         "doctrine/doctrine-bundle": "^1.6",
         "doctrine/doctrine-migrations-bundle": "^1.0",
         "doctrine/orm": "^2.5",
         "eightpoints/guzzle-bundle": "^7.0",
         "leafo/scssphp": "^0.7.1",
-        "mediawiki/oauthclient": "^0.1.2",
         "sensio/framework-extra-bundle": "^5.2",
         "symfony/webpack-encore-pack": "^1.0",
         "symfony/monolog-bundle": "^3.3",
         "symfony/polyfill-apcu": "^1.0",
         "symfony/swiftmailer-bundle": "^3.0",
         "symfony/symfony": "^4.0",
-        "twig/twig": "^2.0"
+        "twig/twig": "^2.0",
+        "wikimedia/toolforge-bundle": "^0.3"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/wikimedia/ToolforgeBundle.git"
```

Comment:
Do we still need this?

Reply:
Nope, not now it's on packagist; I'll update now.